samza-core/src/test/java/org/apache/samza/zk/TestZkJobCoordinator.java

```diff
@@ -200,4 +212,49 @@ public void testShouldStopPartitionCountMonitorWhenStoppingTheJobCoordinator() {
 
     Mockito.verify(monitor).stop();
   }
+
+  @Test
+  public void testStartpointFanOutOnBecomingLeader() throws IOException {
```

Comment:
Is there a test for the `loadMetadataResources` flow? Could the startpoint fanout flow be checked in that test?

Reply:
I'll look into it. Thanks.